src/server/pachyderm_test.go

```diff
@@ -104,6 +105,56 @@ func createTestCommits(t *testing.T, repoName, branchName string, numCommits int
 	}
 }
 
+func deleteEtcd(t *testing.T, ctx context.Context, namespace string) {
```

Comment:
This should call `t.Helper()` so that assertion failures appear from the caller's perspective.  Same with waitForOnePodReady.